.gitignore

```diff
@@ -1,7 +1,9 @@
 build/
 install/
 log/
-/.vscode
+.vscode/
```

Comment:
Things that should be ignored in most projects should be defined in `~/.config/git/ignore` in the home directory. The `.gitignore` from this project should only mention artifacts that are specific to this repo.